### ISSUE TITLE
Json data format for dataset preparation

### ISSUE THREAD

**Issue by gungui98** [emreaksan/deepwriting#13]
Hi, I want to train the model on my own dataset but cannot find your json-typed dataset but the npz instead. So could you pleased to share some of your json data(samples or format)?

**Comment by emreaksan** [comment]
Hey,
I will have a look and try to share at least the data conversion scripts. You can convert json data into numpy and train the model.

**Comment by gungui98** [comment]
yes, I am currently has xml data likes IAMonDB but I don't know how to transfer to json format.

**Comment by gungui98** [comment]
any update yet?

**Comment by namheegordonkim** [comment]
If anyone's wondering in 2024, https://ait.ethz.ch/deepwriting has "extended dataset" which is in JSON.

**Comment by Nouman945** [comment]
hey @namheegordonkim - any luck on running the hand writing beautification?
